stories/Chessboard.stories.tsx

```diff
@@ -123,6 +124,115 @@ export const PlayVsRandom = () => {
   );
 };
 
+///////////////////////////////////
+////////// PlayVsStockfish ////////
+///////////////////////////////////
+
+// 1. Place `stockfish.js` file into React's `publick` folder
+// 2. Then use it as a Web Worker: ``` const stockfish = new Worker("./stockfish.js"); ``` like in `engine.ts`
```

Comment:
comments should be within the components so that users can see them in Storybook